presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -1347,18 +1352,37 @@ public HiveInsertTableHandle beginInsert(ConnectorSession session, ConnectorTabl
         for (PartitionUpdate partitionUpdate : partitionUpdates) {
             if (partitionUpdate.getName().isEmpty()) {
                 // insert into unpartitioned table
+                if (!table.get().getStorage().getStorageFormat().getInputFormat().equals(
```

Comment:
Will do.

I'd like to discuss it a little bit here, I wrap because the line is too long to read, and I find that the presto code are often very very long in one line, it does not annoy you guys? Or because you are all using very big monitor?